easybuild/easyconfigs/l/LAMMPS/LAMMPS-20180316-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,45 @@
+name = 'LAMMPS'
+version = '20180316'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code,
+and an acronym for Large-scale Atomic/Molecular Massively Parallel Simulator."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/lammps/lammps/archive/']
+sources = ['stable_16Mar2018.tar.gz']
+checksums = ['03f91d69f472c37a4008611dee13fad91b6bedc66dbf0393416cdacc751b57ff']
+
+dependencies = [
+    ('FFTW', '3.3.7'),
+    ('Python', '3.6.4'),
+    ('libpng', '1.6.34'),
+    ('libjpeg-turbo', '1.5.3'),
+    ('PCRE', '8.41'),
+    ('libxml2', '2.9.7'),
+    ('zlib', '1.2.11'),
+    ('gzip', '1.8'),
+    ('FFmpeg', '3.4.2'),
+    ('HDF5', '1.10.1'),
+    ('Eigen', '3.3.4', '', True),
+    ('Voro++', '0.4.6'),
+]
+
+# run make in lib/<pkglib> for each of these (and use a particular Makefile, if specified)
+packaged_libraries = ['atc', 'awpmd', 'colvars', 'h5md', 'meam', 'poems', 'qmmm', 'reax']
+
+# run "make yes-<pkg>" for each of these
+packages_yes = ['ASPHERE', 'BODY', 'CLASS2', 'COLLOID', 'COMPRESS', 'CORESHELL', 'DIPOLE', 'GRANULAR',
+                'KSPACE', 'MANYBODY', 'MC', 'MEAM', 'MISC', 'MOLECULE', 'MPIIO', 'PERI', 'POEMS', 'PYTHON',
+                'QEQ', 'REAX', 'REPLICA', 'RIGID', 'SHOCK', 'SNAP', 'SRD', 'USER-ATC', 'USER-AWPMD', 'USER-CG-CMM',
+                'USER-CGDNA', 'USER-COLVARS', 'USER-DIFFRACTION', 'USER-DPD', 'USER-DRUDE', 'USER-EFF', 'USER-FEP',
+                'USER-H5MD', 'USER-LB', 'USER-MANIFOLD', 'USER-MGPT', 'USER-MOLFILE', 'USER-NC-DUMP', 'USER-PHONON',
+                'USER-QMMM', 'USER-QTB', 'USER-REAXC', 'USER-SMD', 'USER-SMTBQ', 'USER-SPH', 'USER-TALLY', 'VORONOI']
+
+# run "make no-<pkg>" for each of these
+packages_no = ['GPU', 'KIM', 'KOKKOS', 'MSCG', 'OPT', 'USER-INTEL', 'USER-MISC', 'USER-OMP', 'USER-QUIP', 'USER-VTK']
```

Comment:
For `LAMMPS` compiled with the `intel` toolchain, is there a reason not to add `USER_INTEL`?

Reply:
Ah, I overlooked that, we should indeed add that one when using Intel compilers...

Maybe we should even have a check for it in the easyblock?

Reply:
Pinging @alikerr for comments here too.

Reply:
@mboisson According to the documentation [1], there should not be any incompatibility with other packages. Plus this is the easyconfig, not the easyblock ;)

[1] http://lammps.sandia.gov/doc/Section_packages.html#user-intel-package

Reply:
I was referring to @boegel's question about checking it in the easyblock.